static_src/stores/org_store.js

```diff
@@ -99,6 +99,12 @@ class OrgStore extends BaseStore {
     return this._currentOrgGuid;
   }
 
+  currentOrgName() {
```

Comment:
@msecret I think this should use the same get accessor patter that `currentOrgGuid` does